pkg/build/webhook/github/github_test.go

```diff
@@ -90,7 +90,7 @@ func TestWrongContentType(t *testing.T) {
 	client := &http.Client{}
 	req, _ := http.NewRequest("POST", server.URL+"/build100/secret101/github", nil)
 	req.Header.Add("Content-Type", "application/text")
-	req.Header.Add("X-Github-Event", "ping")
+	req.Header.Add("X-GitHub-Event", "ping")
```

Comment:
you can change it, we discussed on IRC, header names aren't case sensitive.  but we should be sending it the way github sends it for consistency.